docs/CONTRIBUTING.adoc

```diff
@@ -78,17 +78,17 @@ To create a Polkadot companion PR:
 . Pull latest Polkadot master (or clone it, if you haven't yet).
 . Override your local cargo config to point to your local substrate (pointing to your WIP branch): place `paths = ["path/to/substrate"]` in `~/.cargo/config`.
 . Make the changes required and build polkadot locally.
-. Submit all this as a PR against the Polkadot Repo. Link to your Polkadot PR in the _description_ of your Substrate PR as "polkadot companion: [URL]"
+. Submit all this as a PR against the Polkadot Repo.
+. Link to your Polkadot PR in the _description_ of your _Substrate_ PR as "polkadot companion: [URL]"
+. Link to your Substrate PR in the _description_ of your _Polkadot_ PR as "Companion PR for [URL]"
```

Comment:
that's not necessary, let's avoid cargo cult here
```suggestion
```

Reply:
I noticed that if you didn't do that, then the polkadot CI substrate still pointed to master (but we're told to ignore the polkadot CI so does presumably this is true but doesn't matter?). I think the main point we need to get across is to IGNORE the polkadot CI as it will get run in the substrate PR but maybe things are more nuanced then that?